### DISCUSSION TITLE
How do one listen to deep links?

### DISCUSSION THREAD

**Discussion by erayerdin** [Milad-Akarie/auto_route_library#1679] [Q&A]
`deepLinkBuilder` seems to only work on app launch. Is there a way to listen to it like in [app_links](https://pub.dev/packages/app_links)? It has a `allUriLinkStream` stream.

I test it with `adb` like such:

```plain
adb shell am start -W -a android.intent.action.VIEW -d "appname://app/path?query=param"

**Comment by Milad-Akarie** [comment]
@erayerdin have you tried using deepLinkBuilder inside of router.config()?
